commit 182b13717ae4a2df3e3c6e1f08d562acff57e507
Author:	michalbiesek <michal.biesek@intel.com>
Date:	Mon Jan 28 14:53:40 2019 +0100

    Added new test target
    
    - Added to support test which doesnt' need numa configuration

--- memkind-1.8.0.orig/Makefile.am
+++ memkind-1.8.0/Makefile.am
@@ -429,6 +429,13 @@ static_lib: jemalloc/obj/lib/libjemalloc
 	rm libmemkind.a
 
 
+unit_tests: checkprogs
+	test/all_tests --gtest_filter=*free_with_NULL_kind_4096_bytes:NegativeTest.*:ErrorMessage.*:GetArenaTest.*:MemkindVersioningTests.*
+	test/decorator_test
+
+unit_tests_pmem: checkprogs
+	test/all_tests --gtest_filter=*MemkindPmemTests.*:PmemAllocatorTests.*
+
 include autohbw/Makefile.mk
 include test/Makefile.mk
 include examples/Makefile.mk
